cmd/podman/machine/list.go

```diff
@@ -28,7 +28,7 @@ var (
 		Use:     "list [options]",
 		Aliases: []string{"ls"},
 		Short:   "List machines",
-		Long:    "List Podman managed virtual machines.",
+		Long:    "List machines.",
```

Comment:
@rhatdan @baude, I don't understand why you're reverting this.

Reply:
i believe @rhatdan preferred the more direct use

Reply:
I am fine with the long discription. as managed virtual machines.  Better to have the long description to have more data then the short one.

Reply:
Although I agree with removing Podman.

Reply:
There should be consistency though, so above should be "Initialize a virtual machine"